### DISCUSSION TITLE
HAOS or scripted issue ? MODULE_NOT_FOUND

### DISCUSSION THREAD

**Discussion by hnam2k10** [koush/scrypted#1550] [General]
Hi all,

Some days ago I have updated my scrypted HAOS add-on. I have not checked the cameras right after, and I saw couple of days later that the add on was not starting anymore.
Since there were couple of days between the issue and the update I can't be 100% sure it got broken by the update or not.
This is the scripted log I get when I try to start the add-on:
[scrypted log.txt](https://github.com/user-attachments/files/16579479/scrypted.log.txt)

Looks like I am missing node-gyp-build, any idea how this happened, and how I can go about fixing it ?

Thank you in advance !

**Comment by koush** [comment]
server too old.
